apps/backend/src/modules/quiz/quiz.controller.ts

```diff
@@ -0,0 +1,130 @@
+import { APIPath } from "~/libs/enums/enums.js";
+import {
+	type APIHandlerOptions,
+	type APIHandlerResponse,
+	BaseController,
+} from "~/libs/modules/controller/controller.js";
+import { HTTPCode } from "~/libs/modules/http/http.js";
+import { type Logger } from "~/libs/modules/logger/logger.js";
+
+import { type QuizAnswerService } from "../quiz-answers/quiz-answers.js";
+import { QuizApiPath } from "./libs/enums/enums.js";
+import { type UserDto } from "./libs/types/types.js";
+import { quizUserAnswersValidationSchema } from "./libs/validation-schemas/validation-schemas.js";
+
+/*** @swagger
+ * components:
+ *    schemas:
+ *      UserScore:
+ *        type: object
+ *        properties:
+ *          categoryId:
+ *            type: number
+ *            format: int64
+ *          createdAt:
+ *            type: string
+ *            format: date-time
+ *          score:
+ *            type: number
+ *            format: float
+ *          updatedAt:
+ *            type: string
+ *            format: date-time
+ *          userId:
+ *            type: number
+ *            format: int64
+ *      UserAnswer:
+ *        type: object
+ *        properties:
+ *          answerId:
+ *            type: number
+ *            format: int64
+ *          createdAt:
+ *            type: string
+ *            format: date-time
+ *          id:
+ *            type: number
+ *            format: int64
+ *          updatedAt:
+ *            type: string
+ *            format: date-time
+ *          userId:
+ *            type: number
+ *            format: int64
+ */
+class QuizController extends BaseController {
+	private quizAnswerService: QuizAnswerService;
+
+	public constructor(logger: Logger, quizAnswerService: QuizAnswerService) {
+		super(logger, APIPath.QUIZ);
+
+		this.quizAnswerService = quizAnswerService;
+
+		this.addRoute({
+			handler: (options) =>
+				this.createUserAnswer(
+					options as APIHandlerOptions<{
+						body: { answerIds: number[] };
+						user: UserDto;
+					}>,
+				),
+			method: "PUT",
```

Comment:
Use POST for create and PUT for update

Reply:
This endpoint not just creates, it also deletes previous answers. So that is the reason I use "put" instead of "post" for that